Login/templates/index.html

```diff
@@ -0,0 +1,108 @@
+	<link href="{{ url_for('static' , filename='vendor/animate/animate.css') }}" rel="stylesheet">
+<!--===============================================================================================-->	
+	<link href="{{ url_for('static' , filename='vendor/css-hamburgers/hamburgers.min.css') }}" rel="stylesheet">
+<!--===============================================================================================-->
+    <link href="{{ url_for('static' , filename='vendor/animsition/css/animsition.min.css') }}" rel="stylesheet">	
+<!--===============================================================================================--> 
+	<link href="{{ url_for('static' , filename='vendor/select2/select2.min.css') }}" rel="stylesheet">	
+<!--===============================================================================================-->	
+	<link href="{{ url_for('static' , filename='vendor/daterangepicker/daterangepicker.css') }}" rel="stylesheet">		
+<!--===============================================================================================-->
+	<link href="{{ url_for('static' , filename='css/util.css') }}" rel="stylesheet">
+	<link href="{{ url_for('static' , filename='css/main.css') }}" rel="stylesheet">
+<!--===============================================================================================-->
+</head>
+<body style="background-color: #666666;">
+	
+	<div class="limiter">
+		<div class="container-login100">
+			<div class="wrap-login100">
+				<form class="login100-form validate-form" method="POST" action=/login>
+					<span class="login100-form-title p-b-43">
+						Login to continue
+					</span>
+					
+				
+					<div class="wrap-input100 validate-input" data-validate = "Valid email is required: ex@abc.xyz">
+						<input class="input100" type="text" name="uid">
+						<span class="focus-input100"></span>
+						<span class="label-input100">Username</span>
+					</div>
+					
+					
+					<div class="wrap-input100 validate-input" data-validate="Password is required">
+						<input class="input100" type="password" name="passw">
+						<span class="focus-input100"></span>
+						<span class="label-input100">Password</span>
+					</div>
+
+					<div class="flex-sb-m w-full p-t-3 p-b-32">
+
+					</div>
+			
+
+					<div class="container-login100-form-btn">
+						<button class="login100-form-btn">
+							Login
+						</button>
+					</div>
+				
+
+				</form>
+
+				<div class="login100-more" style="background-image: url('static/images/Money.jpg');">
+				</div>
+			</div>
+		</div>
+	</div>
+	
+	
+
+	
+	
+<!--===============================================================================================-->	
+	<script src="{{ url_for('static' , filename='vendor/jquery/jquery-3.2.1.min.js') }}"></script>   
+
+<!--===============================================================================================-->
+ 
+	<script src="{{ url_for('static' , filename='vendor/animsition/js/animsition.min.js') }}"></script>   
```

Comment:
suggestion no need to implement but probably declaring all the file paths at top would make code more scale able.